wazo_calld/plugins/applications/stasis.py

```diff
@@ -197,15 +201,35 @@ def playback_finished(self, playback, event):
             return
         application = self._service.get_application(application_uuid)
 
-        self._notifier.playback_deleted(application, playback.json)
+        playback_json = playback.json
+        channel_id = None
+        conversation_id = None
+        if playback_json['target_uri'].startswith('channel:'):
+            _, channel_id = playback_json['target_uri'].split('channel:')
+            channel = _ChannelHelper(channel_id, self._ari)
+            conversation_id = channel.conversation_id()
+
+        self._notifier.playback_deleted(
+            application, playback.json, channel_id, conversation_id
+        )
 
     def playback_started(self, playback, event):
         application_uuid = AppNameHelper.to_uuid(event.get('application'))
         if not application_uuid:
             return
         application = self._service.get_application(application_uuid)
 
-        self._notifier.playback_created(application, playback.json)
+        playback_json = playback.json
+        channel_id = None
+        conversation_id = None
+        if playback_json['target_uri'].startswith('channel:'):
+            _, channel_id = playback_json['target_uri'].split('channel:')
```

Comment:
Same thing as previous comment, using `playback.json` instead of creating a temp var to read it
```suggestion
        channel_id = None
        conversation_id = None
        if playback.json['target_uri'].startswith('channel:'):
            _, channel_id = playback.json['target_uri'].split('channel:')
```